samples/azure-quantum/simulators/SparseSimulator.ipynb

```diff
@@ -0,0 +1,2274 @@
+   "metadata": {},
+   "source": [
+    "Now let's run our Q# operation on the sparse simulator, whish is done with the [%simulate_sparse](https://docs.microsoft.com/en-us/qsharp/api/iqsharp-magic/simulate_sparse) magic command."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "empty-commonwealth",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Let's run a big simulation!\n",
+      "Allocating 200 qubits...\n",
+      "Finished allocating!\n",
+      "Creating 2 clusters of GHZ states between 100 qubits...\n",
+      "Finished cluster 0.\n",
+      "Finished cluster 1.\n",
+      "Finished GHZ clusters!\n",
+      "Collapse and show final state:\n",
+      "[One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,One,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero,Zero]\n",
+      "Cleaning up...\n",
+      "Done!\n"
+     ]
+    },
+    {
+     "data": {
+      "application/x-qsharp-data": "{\"@type\":\"tuple\"}",
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%simulate_sparse PrepareAndMeasureGhzState nQubits=200 nClusters=2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "uniform-lesson",
+   "metadata": {},
+   "source": [
+    "From the hystogram above you can see that the sparse simulator stores the non-zero amplitudes only, as opposed to the full state simulator.\n",
+    "\n",
+    "By increasing the `nQubits` parameter you can see that the sparse simulator still provides the result quickly enough, even for the number of qubits where the full state simulator becomes evidently slow.\n",
+    "You can increase the number of qubits up to 1024, which is the current hardcoded limit in the sparse simulator allowing for efficient code."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "excessive-rebate",
+   "metadata": {},
+   "source": [
+    "Now you see that the sparse simulator is an unlocker for users to explore quantum algorithms with large number of qubits but small number of states in superposition."
```

Comment:
I'd again suggest rephrasing this cell in a user-focused way — "unlocker" is kind of tech jargon and isn't common for users that will be reading this sample.